source/adapters/level_zero/command_buffer.cpp

```diff
@@ -754,12 +759,9 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferEnqueueExp(
   // Create a command-list to signal RetEvent on completion
   ur_command_list_ptr_t SignalCommandList{};
   if (Event) {
-    UR_CALL(Queue->Context->getAvailableCommandList(Queue, SignalCommandList,
-                                                    false, false));
-
-    UR_CALL(createEventAndAssociateQueue(Queue, &RetEvent,
-                                         UR_COMMAND_COMMAND_BUFFER_ENQUEUE_EXP,
-                                         SignalCommandList, false));
+    UR_CALL(createEventAndAssociateQueue(
+        Queue, &RetEvent, UR_COMMAND_COMMAND_BUFFER_ENQUEUE_EXP,
+        SignalCommandList, false, false, true));
```

Comment:
It seems that conflict was incorrectly resolved here. getAvailableCommandList should have been left as is. Only createEventAndAssociateQueue should have been changed accordingly.

Please see https://github.com/againull/unified-runtime/commit/f9ac087e2a8fad3b3268f889fa544084715c3ed1#diff-ab021f9451ccb6dd66e3df853d0e123b2f898233a1f609b1e698bef4f6011c71

for the reference.